sdk/documentintelligence/ai-document-intelligence-rest/package.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "@azure-rest/ai-document-intelligence",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "Document Intelligence Rest Client",
```

Comment:
```suggestion
  "description": "Azure Document Intelligence Rest Client",
```
?

Reply:
updated as suggested, but this was generated, TCGC should take care ideally

Reply:
This should be logged as an issue in which repo? @MaryGao